core/modules/system/system.module

```diff
@@ -4278,7 +4278,12 @@ function path_autocomplete($string = '') {
         if ($match_count < $range) {
           $layout_title = '';
           $found_match = FALSE;
-          if (!empty($menu_item['menu']['title']) && stripos($menu_item['menu']['title'], $string) !== FALSE) {
+          $layout = layout_load($menu_item['name'], TRUE);
+          if ($layout && $layout->title) {
```

Comment:
In more modern versions of PHP these non-specific evaluators may cause problems. These should probably be 

`if ($layout != FALSE && !empty($layout->title)) {`